gui/functions.js

```diff
@@ -0,0 +1,30 @@
+function moveCameraXY() {
+  if (camera.position.z == phantom.radius()*1.5) {
+    camera.position.set(0, 0, 0);
```

Comment:
Camera.lookAt(0,0,0) is already enabled when the camera is placed in the scene, so just moving the camera is needed.